app/controllers/users/registrations_controller.rb

```diff
@@ -3,7 +3,7 @@ class Users::RegistrationsController < Devise::RegistrationsController
   before_action :configure_permitted_parameters
 
   def new
-    @page_title = "Create Account" # Displays "New Registration" otherwise
+    @page_subtitle = t("registrations.page_title") # Displays "New Registration" otherwise
```

Comment:
We can take advantage of Rails' [lazy lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup) here and just do `t(".page_title")` for both page subtitles. Then, in the locale file, we can put the following under `users:`:

```
    registrations:
      create:
        page_title: Account Created
      new:
        page_title: Create Account
```

You can also delete the existing code comment on this line -- it was probably useful in the original pull request, but it's not anymore.